spec/android_localize_helper_spec.rb

```diff
@@ -117,12 +117,52 @@ def generated_file(code)
           end
         end
 
+        shared_examples 'en-dash substitutions' do |xpath, fixture_block, gp_block, final_block|
+          it 'has at least one fixture with text to be substituted' do
+            # This ensures that even if we modify the fixtures in the future, we will still have a case which tests this
+            gp_xml = File.open(stub_file('pt-br')) { |f| Nokogiri::XML(f, nil, Encoding::UTF_8.to_s) }
+            gp_node = gp_xml.xpath(xpath).first
+            expect(gp_node.content).to include(fixture_block)
+          end
+
+          TEST_LOCALES_MAP.each do |h|
+            it "has the text substituted in #{h[:android]}" do
+              final_xml = File.open(generated_file(h[:android])) { |f| Nokogiri::XML(f, nil, Encoding::UTF_8.to_s) }
+              final_node = final_xml.xpath(xpath).first
+              expect(final_node.content).to include(final_block)
+              expect(final_node.content).not_to include(gp_block)
+            end
+          end
+        end
+
         context 'with //string tags' do
-          include_examples 'substitutions', "/resources/string[@name='shipping_label_payments_saving_dialog_message']"
+          include_examples 'ellipsis substitutions', "/resources/string[@name='shipping_label_payments_saving_dialog_message']"
+
+          # en-dash: substitute ranges
+          include_examples 'en-dash substitutions', "/resources/string[@name='threat_fix_description']", '0-1', '0-1', '0–1'
```

Comment:
the `shared_examples 'en-dash substitutions'` seems to take 4 parameters, but here you pass 5?

Reply:
Sorry, I don't understand 🤔 Are you sure?

Reply:
Or right, sorry, I counted the name of the example (`en-dash substitutions`) as if it was one of the parameters passed to `|xpath, fixture_block, gp_block, final_block|` param list 🤦  nvm

Reply:
Though, it looks (at least from GitHub) that the last 3 parameters always have the same value each… Is that because from GitHub (and the font they use) the difference between a hyphen and an em-dash is not super visible? If so, would it make sense to use `\u2013` in the parameters that use an em-dash in those examples, to make those test cases more readable?

e.g. something like this (untested)?
```
include_examples 'en-dash substitutions', "/resources/string[@name='threat_fix_description']", '0-1', '0-1', '0\u{2013}1'
```